src/test/groovy/org/prebid/server/functional/tests/DebugSpec.groovy

```diff
@@ -343,4 +346,51 @@ class DebugSpec extends BaseSpec {
         and: "Response should not contain ext.warnings"
         assert !response.ext?.warnings
     }
+
+    def "PBS should emit error when #reason and bidder return error"() {
```

Comment:
You started from testing for errors (or rather their absence), but completely skipped the positive tests for the `seatNonBid`, which should be populated with losing bids and bidders not bidding.